File: week 1/p1_글자 세기/main.java
import java.io.*;
import java.util.Scanner;
class Example {
    public static void main(String[] args) throws Exception {
        Scanner input = new Scanner(System.in);        
        while(input.hasNextLine()) {
            String s = input.nextLine();
            int words = 0, letters = 0;            
            for(int i=0; i < s.length(); i++) {
                if ((i == 0 && (s.charAt(i) != ' ' && s.charAt(i) != '\t')) || i > 0 && (s.charAt(i-1) == ' ' || s.charAt(i-1) == '\t') && (s.charAt(i) != ' ' && s.charAt(i) != '\t'))
                    words++;
                if (s.charAt(i) != '\t' && s.charAt(i) != ' ')
                    letters++;
            }
            System.out.println(letters + " " + words);
        }       
    }
}

File: week 1/p1_글자 세기/main.cpp
#include <iostream>
#include <string>
using namespace std;
int main() {
    string line;
    
    while (getline(cin, line)) {
        int words = 0, letters = 0;
        for (int i = 0; i < line.length(); i++) {
            if ((i == 0 && (line[i] != ' ' && line[i] != '\t')) || (i > 0 && (line[i-1] == ' ' || line[i - 1] == '\t') && (line[i] != ' ' && line[i] != '\t')))
                words++;
            if(line[i] != ' ' && line[i] != '\t')   letters++;
        }
        cout << letters << " " << words << endl;
    }
    return 0;
}
